packages/CamelCase/CamelCase.0.2.0/opam

```diff
@@ -0,0 +1,40 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "A simple OCaml test framework"
+description: "A simple OCaml test framework"
+maintainer: ["Christopher Kaster <me@atomicptr.de>"]
+authors: ["Christopher Kaster <me@atomicptr.de>"]
+license: "MIT"
+tags: ["tests" "test" "unit" "unit tests"]
+homepage: "https://github.com/atomicptr/CamelCase"
+doc: "https://github.com/atomicptr/CamelCase"
+bug-reports: "https://github.com/atomicptr/CamelCase/issues"
+depends: [
+  "ocaml"
```

Comment:
This should address the [failing tests on older ocaml versions](https://opam.ci.ocaml.org/github/ocaml/opam-repository/commit/5935cebe8f575fe2d136afe61b8732d39349390b):

```suggestion
  "ocaml" {>= "4.08"}
```